package nom.brunokarpo.reviewworker.service.exceptions

class ReviewDuplicatedException : Throwable()
